frontend/src/app/modules/home/projects/edit/project-general/project-general.component.html

```diff
@@ -7,7 +7,7 @@
         <div class="item-field">
             <div class="item-form-input p-input-icon-right">
                 <i *ngIf="name?.pending" class="pi pi-spin pi-spinner"></i>
-                <input formControlName="name" pInputText placeholder="Project name" class="item-form-field">
+                <input formControlName="name" [maxLength]="50" pInputText placeholder="Project name" class="item-form-field">
```

Comment:
Why are you use this instead of validation inside ts file. If you are using reactive forms, please move this validations inside control adding